db/view/row_locking.cc

```diff
@@ -80,7 +80,7 @@ row_locker::lock_pk(const dht::decorated_key& pk, bool exclusive, db::timeout_cl
     // even in the case of rehashing.
     return f.then([this, pk = &i->first, exclusive, &single_lock_stats, waiting_latency = std::move(waiting_latency)] () mutable {
         waiting_latency.stop();
-        single_lock_stats.estimated_waiting_for_lock.add(waiting_latency.latency(), single_lock_stats.operations_currently_waiting_for_lock);
+        single_lock_stats.estimated_waiting_for_lock.add(waiting_latency.latency());
```

Comment:
What is lost by not passing that parameter?

Reply:
It's no longer supported. Originally, the idea was to sample the data, so when adding a value to the histogram it was added with the new total number of values (you can see that the code here just adds n+1)

Because all the values are added to the histogram there is no need for all those manipulations